### PR TITLE
fix: footer display "2021 ~ 2021"

### PR DESCRIPTION
**Pull request by cnrkuo**

### CODE REVIEW

**Comment by kuohuanhuan** [comment]
RFRV

**Review by cnrkuo — COMMENTED** [review]
Ready.

File: layout/_partial/footer.ejs [hunk 1]
```diff
@@ -5,9 +5,14 @@
...
           Copyright ©
           <% if(theme.footer.copyYear) { %>
-            <%- theme.footer.copyYear %> -
+            <%- theme.footer.copyYear %>
+              <% if (theme.footer.copyYear != date(Date.now(), 'YYYY'))
+                -
+              <% } %>
+          <% } %>
+          <% if (theme.footer.copyYear != date(Date.now(), 'YYYY'))
+                <%- date(Date.now(), 'YYYY') %>
```
Comment (songhn233) [inline layout/_partial/footer.ejs:14]:
```suggestion
            <%- theme.footer.copyYear %>
          <% } %>
          <% if (theme.footer.copyYear != date(Date.now(), 'YYYY')) { %>
                - 
                <%- date(Date.now(), 'YYYY') %>
```


`if` can be merged.

**Comment by songhn233** [comment]
Thanks PR!

**Pull request merged**